frameworks/kafka/README.md

```diff
@@ -1,14 +1,420 @@
-# DC/OS Apache Kafka Service Documentation
+<p align="left"><img src="https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/dcos-sdk-logo.png" width="250"/></p>
+
+
+# Apache Kafka Service Guide
 
 ## Table of Contents
 
-- [Overview](docs/index.md)
-- [Quick Start](docs/quick-start.md)
-- [Install and Customize](docs/install-and-customize.md)
-- [Uninstall](docs/uninstall.md)
-- [Configure](docs/configure.md)
-- [Connecting Clients](docs/connecting-clients.md)
-- [Managing](docs/managing.md)
-- [API Reference](docs/api-reference.md)
-- [Troubleshooting](docs/troubleshooting.md)
-- [Limitations](docs/limitations.md)
+- [Overview](#overview)
+  - Features
+- [Quick Start](#quick-start)
+- [Installing and Customizing](#installing-and-customizing)
+  - Installation with Default Settings
+  - Installation with Custom Settings
+  - Service Settings
+    - Service Name
+    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
```

Comment:
Should update the TOC to reflect the content below. For example some of these suggested sections can be cleaned up, and 'node settings' was renamed to 'broker settings'